temporalcli/commandsgen/commands.yml

```
@@ -287,6 +288,81 @@ commands:
     option-sets:
       - workflow reference
 
+
+  - name: temporal activity update
```

```suggestion
  - name: temporal activity update-options
```

Can we change this to `update-options`? I am afraid of a future concept of an "activity update" that is unrelated to updating options